packages/styles/src/core/layout/README.md

```diff
@@ -40,24 +39,30 @@ collapsed: true
 ### Toolbar
 Toolbar is a generic bar that can be used as a header to put your application logo and some options to navigate throughout your application.
 
-The toolbar is placed in the top of your application unless you modify its position by using `left` or `right` modifiers in `airship-app` styles.
+On mobile devices the toolbar is placed always on top but its position can be modified on desktop by using `left` or `right` modifiers in `airship-app` styles.
   - `airship-app--nav-left`: Push the toolbar to the left of the viewport.
   - `airship-app--nav-right`: Push the toolbar to the right of the viewport.
 
 ### Toolbar Actions
 
-Your toolbar can contain several buttons to trigger actions in your application. These actions need to be grouped in an element with `.as-toolbar-actions` class.
+Toolbar actions are the main toolbar's content, usually icons or links to another app's section. Actions are hidden by default on mobile devices and they are only visible when the `as-toolbar-actions--visible` modifier class is present.
+
+In order to [fit the W3C Accessibility guidelines](https://www.w3.org/WAI/tutorials/menus/) the menu should be an `ul` element where menu item should be a `li` element with a `as-toolbar__item` class.
 
 ```code
 lang:html
 ---
-<nav class="as-toolbar-main">
-  <div class="as-toolbar-actions">
-    <span class="as-toolbar-main__item">
-      <img src="https://material.io/tools/icons/static/icons/baseline-fingerprint-24px.svg" alt="Ajustes">
-      <p>Ajustes</p>
-    </span>
-  </div>
+<nav class="as-toolbar__actions">
+  <ul>
+    <li class="as-toolbar__item">
+      <img src="https://material.io/tools/icons/static/icons/baseline-fingerprint-24px.svg" alt="Action 1">
+      <p class="as-toolbar__icon-text">Action 1</p>
+    </li>
+    <li class="as-toolbar__item">
+      <img src="https://material.io/tools/icons/static/icons/baseline-add_location-24px.svg" alt="Action 2">
+      <p class="as-toolbar__icon-text">Action 2</p>
+    </li>
+  </ul>
 </nav>
 ```
 
```

Comment:
I'm missing an explanation of what `openSideMenu()` does. You tell that you need a button but you also need a event handler that does something, right? If you only provide a button, nothing will happen.

Reply:
`as-toolbar-main` is outdate throughout all code examples.